### DISCUSSION TITLE
Workspace switching keybinds?

### DISCUSSION THREAD

**Discussion by WaddlesPlays** [Floorp-Projects/Floorp#695] [Feature Request]
I recently started using the workspaces option, but I cannot find or figure out the keybinds for switching workspaces. I would like to be able to set shortcuts to switch to the next/previous workspace so I don't have to drag my mouse all the way over to the toolbar button or the sidebar to switch them. This would make navigation between workspace much easier and accessible, and likely make the feature more usable in minimal userChrome configs that discourage toolbar clutter like one-line themes.

**Comment by WaddlesPlays** [comment]
I'm on 11.8.2 now and I still don't see anything like "Next/Prev Workspace" in about:preferences#cSK, I couldn't brute-force anything from trying random shortcuts, and searching around for other discussions hasn't yielded any other results. I'm changing this into a feature request because at this point I'm convinced it's just not implemented yet.

**Reply by CutterKnife** [reply]
https://www.reddit.com/r/Floorp/comments/19cn4ft/workspace_shortcuts/

**Comment by WaddlesPlays** [comment]
Can confirm the release of 11.9.0 added options for this in about:preferences:cSK! It's a bummer I can't bind with any key that isn't a letter or number but issues opened up about it seem to mention an upstream issue. Thanks for implementing this!

**Reply by Ryosuke-Asano** [reply]
I forgot to write it to Releasenots
